lms/djangoapps/certificates/signals.py

```diff
@@ -102,6 +101,8 @@ def _listen_for_id_verification_status_changed(sender, user, **kwargs):  # pylin
         return
 
     user_enrollments = CourseEnrollment.enrollments_for_user(user=user)
+
```

Comment:
Same question as the previous comment: Why import this here instead of at the top level?  Can you add a comment if there is a good reason?